File: app/src/main/java/jogja/telkom/csrassistant/AddOn/AddOnActivity.java
package jogja.telkom.csrassistant.AddOn;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import jogja.telkom.csrassistant.R;

public class AddOnActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_addon);

        Button tambah = findViewById(R.id.tambah_addon);
        Button cabut = findViewById(R.id.cabut_addon);

        tambah.setOnClickListener(v ->
                startActivity(new Intent(AddOnActivity.this,TambahAddOnActivity.class)));

        cabut.setOnClickListener(v ->
                startActivity(new Intent(AddOnActivity.this,CabutAddOnActivity.class)));

    }

}

File: app/src/main/java/jogja/telkom/csrassistant/AddOn/AddOnActivity.kt
package jogja.telkom.csrassistant.AddOn

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import jogja.telkom.csrassistant.R

class AddOnActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_addon)

        val tambah = findViewById<Button>(R.id.tambah_addon)
        val cabut = findViewById<Button>(R.id.cabut_addon)

        tambah.setOnClickListener { v -> startActivity(Intent(this@AddOnActivity, TambahAddOnActivity::class.java)) }

        cabut.setOnClickListener { v -> startActivity(Intent(this@AddOnActivity, CabutAddOnActivity::class.java)) }

    }

}
